templates/Engine method templates/CreateTemplate/CreateTemplate.cs

```diff
@@ -21,34 +21,39 @@
  */
 
 using System;
-using System.Collections;
-using System.Collections.Generic;
 using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
+using System.Collections.Generic;
+using System.ComponentModel;
+using BH.oM.Reflection.Attributes;
 using BH.oM.Base;
-using BH.oM.Structure.Elements;
-using BH.oM.Structure.Constraints;
 
-namespace BH.Adapter.$ext_safeprojectname$
+namespace BH.Engine.External.SoftwareName // Replace SoftwareName with the name of your Toolkit software.
```

Comment:
Namespace as above.

`namespace $rootnamespace$` with comment about removing 'Create' if it exists I think.